### ISSUE TITLE
New version of gcloud-aio-auth doesn't works properly

### ISSUE THREAD

**Issue by froosty** [talkiq/gcloud-aio#167]
Related to PR #166 and issue #126 

After this update library works incorrect with aiohttp 3.6.*

When you call [raise_for_status](https://github.com/talkiq/gcloud-aio/blob/master/auth/gcloud/aio/auth/session.py#L66) method from original library, response will be [automatically released](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/client_reqrep.py#L896) in case of "bad" status code. This means that you can't read body after this call. You'll have `aiohttp.ClientConnectionError` [from here](https://github.com/aio-libs/aiohttp/blob/master/aiohttp/streams.py#L334)

Code to reproduce:
```python
import asyncio
import aiohttp


# This function was just copied from your library
async def _raise_for_status(resp):
    try:
        resp.raise_for_status()
    except aiohttp.ClientResponseError:
        await resp.text()
        raise


async def main():
    async with aiohttp.ClientSession() as session:
        resp = await session.delete('http://example.com/non-existing-url')

        try:
            await _raise_for_status(resp)
        except aiohttp.ClientResponseError:
            print('Expected behaviour')
        except aiohttp.ClientConnectionError:
            print('Unexpected behaviour')


asyncio.run(main())

```

I can make a PR (if needed) but you should choose approach

As for me the only way to retrieve body from response in this case - to read the body from response before call `resp.raise_for_status()`. But this is a bad approach to read the body inside library only for additional logging, IMHO.

In any case current functionality doesn't works properly.

**Comment by TheKevJames** [comment]
Ah! Thanks for catching this, I completely missed that. Will have a fix and a new bugfix release up shortly.

**Comment by TheKevJames** [comment]
Fixed in 4306c97b74521feea940c7df894b90ccefa786a5 and will shortly be released as `gcloud-aio-auth==3.2.1`. Thanks again!
